qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
Hi Pedro, please, just make it right, I do not need to know the theory, I trust you ;)
I understood that language code "pt" correspond to locale "pt_PT", while code "pt-br" defines "pt_BR".

If I search for "pt-pt" I get nothing in -X, that is why line "pt-pt by Pedro Mendonça" is not helpful. Would this be correct:
	pt by netolazaro
	pt-br by Pedro Mendonça
or maybe this
	pt by netolazaro, Pedro Mendonça
	pt-br by Pedro Mendonça